### PR TITLE
Nov16 travis workspaces

### PR DESCRIPTION
**Pull request by IzaacAyelin**

### CODE REVIEW

File: packages/gallery/package.json [hunk 1]
```diff
@@ -25,14 +25,15 @@
...
     "test": "jest --testPathIgnorePatterns '.*/e2e/.*.e2e.spec.js' --silent",
     "posttest": "yoshi lint",
     "release": "yoshi release",
     "lint": "eslint 'src/**/*.{js,jsx}'",
     "lint:fix": "npm run lint -- --fix",
     "start-with-test": "PORT=9837 yoshi start --with-test",
-    "e2e-test": "jest --testPathPattern='.*/e2e/.*.e2e.spec.js'"
+    "e2e-test": "jest --testPathPattern='.*/e2e/.*.e2e.spec.js'",
+    "e2e-layouts": "jest --testPathPattern='.*/e2e/layouts/.*.e2e.spec.js'",
+    "e2e-styleParams": "jest --testPathPattern='.*/e2e/styleParams/.*.e2e.spec.js'"
```
Comment (IzaacAyelin) [inline packages/gallery/package.json]: add integration-e2e tests that Yonatan added

File: packages/gallery/package.json [hunk 2]
```diff
@@ -25,14 +25,15 @@
...
     "precommit": " ",
-    "pretest": "yoshi build",
     "test": "jest --testPathIgnorePatterns '.*/e2e/.*.e2e.spec.js' --silent",
     "posttest": "yoshi lint",
     "release": "yoshi release",
     "lint": "eslint 'src/**/*.{js,jsx}'",
     "lint:fix": "npm run lint -- --fix",
     "start-with-test": "PORT=9837 yoshi start --with-test",
-    "e2e-test": "jest --testPathPattern='.*/e2e/.*.e2e.spec.js'"
+    "e2e-test": "jest --testPathPattern='.*/e2e/.*.e2e.spec.js'",
```
Comment (IzaacAyelin) [inline packages/gallery/package.json]: rename to test:e2e:all

File: package.json [hunk 3]
```diff
@@ -17,12 +17,13 @@
...
+    "test:e2e": "start-server-and-test build-all-and-start-playground http-get://localhost:3000 'lerna run --parallel test:e2e-all'",
     "test": "npm run link-packages && npm run build-packages-by-order && lerna run --parallel test",
-    "start-playground:ci": "cd packages/playground && npx serve -s build -l 3000 && cd ../..",
-    "test:ci": "npm run link-packages && lerna run --parallel test",
-    "test:e2e:layouts:ci": "npm run link-packages && start-server-and-test start-playground:ci http-get://localhost:3000 'lerna run e2e-layouts'",
-    "test:e2e:styleParams:ci": "npm run link-packages && start-server-and-test start-playground:ci http-get://localhost:3000 'lerna run e2e-styleParams'",
+    "ci:run-playground:": "cd packages/playground && npx serve -s build -l 3000 && cd ../..",
+    "ci:test-unit": "npm run link-packages && lerna run --parallel test",
+    "ci:test-e2e-layouts": "npm run link-packages && start-server-and-test run-playground:ci http-get://localhost:3000 'lerna run test:e2e-layouts'",
+    "ci:test-e2e-styleParams": "npm run link-packages && start-server-and-test run-playground:ci http-get://localhost:3000 'lerna run test:e2e-styleParams'",
```
Comment (IzaacAyelin) [inline package.json]: fixed **run-playground:ci** to **ci:run-playground** in the next commit

File: .travis.yml [hunk 4]
```diff
@@ -35,21 +35,27 @@ jobs:
...
       workspaces:
         use: dev_build
       name: "E2E Tests - styleParams"
       script:
-        - npm run test:e2e:styleParams:ci || travis_terminate 1;
+        - npm run ci:test-e2e-styleParams || travis_terminate 1;
+    - stage: Tests
+      workspaces:
+        use: dev_build
+      name: "E2E Tests - styleParams"
```
Comment (ritabazak) [inline .travis.yml]: change the name to "E2E Tests - integration"

File: package.json [hunk 5]
```diff
@@ -17,12 +17,13 @@
     "start-playground": "cd packages/playground && npm run start",
     "build-all-and-start-playground": "npm run link-packages && npm run build-packages-by-order && npm run start-playground",
     "build:dev": "npm run link-packages && npm run build-packages-by-order",
-    "test:e2e": "start-server-and-test build-all-and-start-playground http-get://localhost:3000 'lerna run --parallel e2e-test'",
+    "test:e2e": "start-server-and-test build-all-and-start-playground http-get://localhost:3000 'lerna run --parallel test:e2e-all'",
```
Comment (ritabazak) [inline package.json]: can remove --parallel

**Pull request merged**